src/pages/commits/CommitsWrapper.tsx

```diff
@@ -42,22 +41,13 @@ export const CommitsWrapper: React.FC<Props> = ({
         <FlexRowContainer numCommits={versions.length}>
           {versions.map(({ version, rolledUpVersions }) =>
             version ? (
-              <ColumnContainer key={version.id}>
-                <CommitChart
-                  groupedTaskStats={
-                    idToTaskStatsGroupedByColor[version.id].stats
-                  }
-                  total={idToTaskStatsGroupedByColor[version.id].total}
-                  max={max}
-                  chartType={chartType}
-                />
-                <CommitChartLabel
-                  githash={version.revision.substring(0, 5)}
-                  createTime={version.createTime}
-                  author={version.author}
-                  message={version.message}
-                />
-              </ColumnContainer>
+              <ActiveCommit
+                version={version}
+                chartType={chartType}
+                total={idToTaskStatsGroupedByColor[version.id].total}
```

Comment:
nit: renamed this to `versionToTaskStatColorMap`?

Reply:
renamed to `versionToGroupedTaskStatsMap` since it is `{ [id: string]: GroupedResult;}` :)